easybuild/easyconfigs/p/pymatgen/pymatgen-4.1.1-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = "PythonPackage"
+
+name = 'pymatgen'
+version = '4.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/pymatgen'
+description = """Python Materials Genomics is a robust materials analysis code that defines core object
+ representations for structures and molecules with support for many electronic structure codes."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('spglib-python', '1.9.4.2', versionsuffix),
```

Comment:
@boegel that `_spglib.so` is built from the C sources in the downloaded package, that can be seen in the easybuild log

the python code calls that _spglib for everything, and it is working, e.g.

> import spglib
> silicon = ([(4, 0, 0), (0, 4, 0), (0, 0, 4)],
>                [(0, 0, 0), (0, 0.5, 0.5), (0.5, 0, 0.5), (0.5, 0.5, 0), 
                   (0.25, 0.25, 0.25), (0.25, 0.75, 0.75), (0.75, 0.25, 0.75), (0.75, 0.75, 0.25)],
>                 [14,] * 8)
> print spglib.get_spacegroup(silicon)

outputs "Fd-3m (227)" (which is correct :) )